charts/stable/openemr/values.yaml

```
@@ -6,48 +6,56 @@
 #
 
 image:
+  # -- image repository
   repository: openemr/openemr
+  # -- image pull policy
   pullPolicy: IfNotPresent
-  tag: 6.0.0
+  # -- image tag
+  tag: 6.1.0
 
-strategy:
-  type: Recreate
-
-# See more environment variables in the openemr documentation
-# https://github.com/openemr/openemr/blob/master/docker/production/docker-compose.yml
-env: {}
-  # TZ:
+# -- environment variables. See more environment variables in the [openemr documentation](https://github.com/openemr/openemr/blob/master/docker/production/docker-compose.yml)
+# @default -- See below
+env:
+  # -- Set the container timezone
+  TZ: UTC
   # MYSQL_HOST: openemr-mariadb
   # MYSQL_ROOT_PASS: openemrrootpass
   # MYSQL_USER: openemr
   # MYSQL_PASS: openemr-pass
   # OE_USER: admin
   # OE_PASS: pass
 
+# -- Configures service settings for the chart.
+# @default -- See values.yaml
 service:
-  port:
-    port: 80
-  additionalPorts:
-    - name: websocket
-      port: 443
+  main:
+    primary: true
+    ports:
+      http:
+        port: 80
```

```suggestion
        port: 80
        primary: true
```

I suggest making one of these to the explicit primary port (that will also be used for the liveness probe, etc). I don't know the app well enough to say if that should be the `http` or `https` port though.